docs/cn/Configuring-Alluxio-with-Ceph.md

```diff
@@ -18,18 +18,13 @@ priority: 1
 
 首先，在你的机器上必须安装Alluxio二进制包。你可以自己[编译Alluxio](Building-Alluxio-Master-Branch.html)，或者[下载二进制包](Running-Alluxio-Locally.html)
 
-接着，如果还没有配置，请用‘bootstrapConf'命令创建自己的配置。
-例如，如果你在本地机器运行Alluxio，就需要将`ALLUXIO_MASTER_HOSTNAME`设置为`localhost`
-
-{% include Configuring-Alluxio-with-GCS/bootstrapConf.md %}
-
-或者，您也可以从模板创建配置文件并手动设置内容。
-
-{% include Common-Commands/copy-alluxio-env.md %}
-
 ## 配置Alluxio
 
-为了配置Alluxio以使用Ceph作为其底层文件系统，需要修改`conf/alluxio-env.sh`文件
+为了配置Alluxio以使用底层文件系统，需要修改`conf/alluxio-env.sh`文件。如果该文件不存在，根据模板创建配置文件。
```

Comment:
conf/alluxio-env.sh -> alluxio-site.properties

Reply:
Thanks for your comment and I have changed this mistake